### ISSUE TITLE
where english bruh

### ISSUE THREAD

**Issue by ugrolol** [MalitsPlus/ShizuruNotes#22]
![image](https://user-images.githubusercontent.com/36813282/86296029-870cb400-bc00-11ea-82fa-11e369411bd7.png)
i was thinking that there no translation but it exist just doesnt work on russian system language

**Comment by ugrolol** [comment]
![image](https://user-images.githubusercontent.com/36813282/86296168-d9e66b80-bc00-11ea-933f-23c8762e25b0.png)
also ye thing about sorting (its changed on main character list but not in comparison)

**Comment by ugrolol** [comment]
when making a build im not sure why but it showing that
ShizuruNotes\app\build\intermediates\compile_and_runtime_not_namespaced_r_class_jar\release\R.jar: 
busy with another process so cant get access to file

**Comment by MalitsPlus** [comment]
I switched the primary language of my device to Russian but cannot reproduce this issue. Does your language option don't work on any system language or just Russian? 
And thanks for making me notice the font size problem, I will fix this later. 

![image](https://user-images.githubusercontent.com/50281195/86310634-1c379900-bc51-11ea-8021-1c2aaf400359.png)
![image](https://user-images.githubusercontent.com/50281195/86310640-248fd400-bc51-11ea-8622-6436fb089f6e.png)

**Comment by ugrolol** [comment]
ye it happens on any other language too and on emu device in android studio too
cos when app doesnt find out russian locale (or any other language that doesnt exist) it just use default
i actually thought that you aware about that cos i was annoyed by this from the start and thats why was building my own thing

check there vid https://yadi.sk/i/HSTuT9Wy-ya52g
also there should be problem with wording sometimes like **nearest enemy three** in my case i just changed order in targetparameter.java i have no idea how cn language works so idk how you can easy change stuff like that for one language without changing whole app
thats reason why i stopped thinking about russian translation cos russians not that smart to use apps in first place and english is more then fine in this case and when you translate its just too complicated 
![image](https://user-images.githubusercontent.com/36813282/86369775-9125c580-bc87-11ea-92d5-f5c446d36579.png)

i didnt checked text itself
(like im p sure wording in my version maybe better and things like actually noticable )
![image](https://user-images.githubusercontent.com/36813282/86369481-2d9b9800-bc87-11ea-8319-8b57063eb15e.png)

right now basically all changes that i need to make it usable for me
![image](https://user-images.githubusercontent.com/36813282/86369247-d39ad280-bc86-11ea-9f74-2b32e36b2543.png)

**Comment by MalitsPlus** [comment]
Ah, I think I've figured out the reason according to your video capture. 
Generally, ShizuruNotes **Application** will be relaunched after switching language due to [this line](https://github.com/MalitsPlus/ShizuruNotes/blob/02c2e542639fe4dd0790e81c1ed3117673e6b066/app/src/main/java/com/github/malitsplus/shizurunotes/ui/setting/SettingFragment.kt#L90), but it seems that ShizuruNotes just restarted the **Activity** in your video. 
It works well on my device but I don't know whether there are any permission settings or any other stuffs prevent it relaunching on your device. 
For more details your can refer this issue[#14].

**Comment by ugrolol** [comment]
i mean did you try it on emu?
maybe its just working only on your device as you expected

**Comment by MalitsPlus** [comment]
I tried it on my two devices within Android 8 and 10 but emulator. Since there are thousands of customized Android operating systems I have no power to test them all. 
And I'm not sure whether there would be any new troubles if I implement a non-stable appcompat so I decided to wait for a stable version. 
Btw have you tried 
switch language → close app manually → reopen app 
I wonder whether language can be switched normally this way.
